### DISCUSSION TITLE
Missing values

### DISCUSSION THREAD

**Discussion by faelenor** [emporia-vue-local/esphome#81] [Q&A]
One of my circuits started having periods where no values are sent. On the screenshot, the data is okay until 14:30 but then, there's a lot of missing values until 21:00. (I used lines instead of dots in Grafana so we can clearly see the data).

![image](https://user-images.githubusercontent.com/10888149/204363527-0d6ebe97-3bda-4661-bd5c-d1338833dcac.png)

This is my water pump circuit, so it should only have short ~30s spikes of power. What could cause missing values like this?
Thanks!

**Comment by faelenor** [comment]
I saw that I was on an old version of the emporia_vue esphome component (early April). Anyone knows if there was a bug that could cause this? I upgraded my firmware, maybe the issue will disappear...

**Reply by flaviut** [reply]
I'm unaware of any bug that could cause this... hopefully the upgrade takes care of things. If not, a few follow-up questions:

Do you see the same issues with other sensors on the vue as well? could it be a wifi connection issue?

If it doesn't fix itself, you can leave [the ESPHome logs running](https://esphome.io/guides/cli.html#logs-command) in the background to see if there are any hints in there about what's wrong.

**Reply by faelenor** [reply]
It's still doing the same thing. I will have a look at the connector as copekyle suggested, it may be the issue.

**Comment by copekyle** [comment]
I was seeing this on one of my circuits and it ended up that the plug on the port had a little strain on it and over time, it stopped making good connection. The plug itself didn't look like it was seated incorrectly. I would unplug the port and plug it back in and see if it goes away.

**Reply by faelenor** [reply]
Hmm, I will check this. I thought that the device would still send data (zeros) even if the connector was not properly connected.
I always have issues with these connectors, they disconnect so easily. I don't understand why they used them, it's so frustrating.
